src/main/kotlin/lotto/data/LottoRanking.kt

```diff
@@ -0,0 +1,33 @@
+package lotto.data
+
+enum class LottoRanking(val matchingNumberCnt: Int, val price: Int) : Prize {
+    FirstPlace(6, 2000000000) {
+        override fun findPrize(winningStatus: Pair<LottoRanking, Int>): Int {
+            return winningStatus.first.price * winningStatus.second
+        }
+    },
+    SecondPlace(5, 1500000) {
+        override fun findPrize(winningStatus: Pair<LottoRanking, Int>): Int {
+            return winningStatus.first.price * winningStatus.second
+        }
+    },
+    ThirdPlace(4, 50000) {
+        override fun findPrize(winningStatus: Pair<LottoRanking, Int>): Int {
+            return winningStatus.first.price * winningStatus.second
+        }
+    },
+    FourthPlace(3, 5000) {
+        override fun findPrize(winningStatus: Pair<LottoRanking, Int>): Int {
+            return winningStatus.first.price * winningStatus.second
+        }
+    },
+    None(0, 0) {
+        override fun findPrize(winningStatus: Pair<LottoRanking, Int>): Int {
+            return winningStatus.first.price * winningStatus.second
+        }
+    };
+}
+
+interface Prize {
```

Comment:
Prize라는 인터페이스를 선언해서 로직을 강제하셨네요 😀
하나의 기능만을 제공하는 인터페이스(Single Abstract Interface, SAM)라면 다음과 같이 fun 을 붙혀서 제공할 수 있어요 자바의 FunctaionInterface애노테이션과 같은 역할을 합니다. 이 경우 해당 인터페이스의 함수를 람다식으로 작성할 수 있다는 장점이 있죠 ㅎㅎ
```suggestion
fun interface Prize {
```

참고: https://flow9.net/bbs/board.php?bo_table=kotlin&wr_id=20&device=pc